com.github.treagod.spectator.json

```diff
@@ -0,0 +1,65 @@
+{
+  "app-id": "com.github.treagod.spectator",
+  "base": "io.elementary.BaseApp",
+  "base-version": "juno-20.08",
+  "runtime": "org.gnome.Platform",
+  "sdk": "org.gnome.Sdk",
+  "runtime-version": "3.38",
+  "command": "com.github.treagod.spectator",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--talk-name=org.gnome.SettingsDaemon"
```

Comment:
Is it needed for anything? I took a quick look at the code and it doesn't seem used anywhere.

Reply:
Actually looks like something is bugged although maybe only on my system? Added a GET request to Flathub API but nothing happens after clicking on "play" icon. I see an error when I pass `--log-session-bus` to flatpak:

```
❯ flatpak run --log-session-bus com.github.treagod.spectator
(...)
C33: -> :1.89 call org.freedesktop.DBus.Properties.GetAll at /org/freedesktop/portal/desktop
B830: <- :1.89 return from C33
C34: -> :1.89 call org.freedesktop.portal.ProxyResolver.Lookup at /org/freedesktop/portal/desktop
B832: <- :1.89 return error org.freedesktop.portal.Error.NotAllowed from C34
```

Reply:
Actually I can't make any sense of the error message. Is there something I can try to solve the problem?

Reply:
First of all, does it work for you? Maybe the issue is on my end and everything is fine with the app.